cdc/sink/codec/canal_flat_test.go

```diff
@@ -337,6 +337,18 @@ func (s *canalFlatSuite) TestEncodeCheckpointEvent(c *check.C) {
 	}
 }
 
+//var testCaseInsert = &model.RowChangedEvent{
+//	CommitTs: 429457251303161859,
+//	Table: &model.TableName{
+//		Schema: "cdc_tiflash_test",
+//		Table:  "multi_data_type",
+//	},
+//	Columns: []*model.Column{
+//		{Name: "id"},
+//	},
+//	PreColumns: nil,
+//}
```

Comment:
Could you uncomment or remove?

Reply:
The PR is not ready to be reviewed.
this is a very important test case which will be filled in later.